media/css/mozorg/home/home-promo.less

```diff
@@ -791,33 +769,11 @@ html[lang|="en"] {
     }
 }
 
-/****************************************************************************/
-// Appmaker
-.promo-small-landscape.appmaker {
-    background-color: #5e6ba1;
-    background-image: url(/media/img/home/voices/promos/appmaker/appmaker-logo.png);
-    background-position: 5px 15px;
-    background-repeat: no-repeat;
-    .background-size(78px, 100px);
-
-    h2 {
-        padding: 10px 10px 10px 95px;
-    }
-}
-
-html[lang|="en"] {
-    .promo-small-landscape.appmaker {
-        background-position: 15px 15px;
-        h2 {
-            padding: 34px @baseLine @baseLine 120px;
-        }
-    }
-}
 
 /****************************************************************************/
 // Volunteer
 .promo-small-landscape.volunteer {
-    background-color: #56565a;
+    background: #0ad; // bright blue
```

Comment:
Any reason not to use shorthand for the whole thing?
`background: #0ad url(/media/img/home/voices/promos/volunteer/volunteer.png) 15px 25px no-repeat;`